README.md

```diff
@@ -41,10 +41,28 @@ in Internet Explorer on Windows 7 for instance (because of the wrong MIME type).
 
 JavaScript Cookie supports [npm](https://www.npmjs.com/package/js-cookie) and [Bower](http://bower.io/search/?q=js-cookie) under the name `js-cookie`.
 
+- #### Bower
+  ```
+  bower install js-cookie --save
+  ```
+
+- #### NPM
+  ```
+  npm install js-cookie --save
```

Comment:
I'm not sure if using `npm` specific command example will help. People might be using yarn.